src/desktop/apps/auth/routes.coffee

```diff
@@ -8,9 +8,12 @@ sanitizeRedirect = require '@artsy/passport/sanitize-redirect'
 @resetPassword = (req, res) ->
   if req.query.reset_password_token?
     req.session.reset_password_token = req.query.reset_password_token
+    req.session.set_password = req.query.set_password
+    req.session.redirect_to = req.query.redirect_to
     res.redirect '/reset_password'
   else
-    res.render 'reset_password', reset_password_token: req.session.reset_password_token
+    res.locals.sd.REDIRECT_TO = req.session.redirect_to
```

Comment:
Maybe a more specific name for this? Something like `RESET_PASWORD_REDIRECT_TO` or something.